libraries/logging/stream-logger-android/src/main/java/io/getstream/logging/android/file/impl/ActivityLifecycleCallbacks.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.logging.android.file.impl
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+
+internal abstract class ActivityLifecycleCallbacks : Application.ActivityLifecycleCallbacks {
```

Comment:
Do we really need this abstract class? :thinking:

Reply:
Wanna keep `LifecycleAwareLogFileManager` clean from redundant functions